source/usage-examples/updateOne.txt

```diff
@@ -0,0 +1,32 @@
+=================
+Update a Document
+=================
+
+.. default-domain:: mongodb
+
+You can update a single document using the `updateOne() <https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne>`_ method. 
+``updateOne()`` takes a filter for a document, and updates the first document
```

Comment:
Perhaps we should use the same terminalogy across all of our methods in the docs. What do we say the other find methods in this position for the action performed?

Reply:
I think you're specifically referring to `query` vs `filter`. I agree that they need to be standard rather than synonyms. Defining which one to use is the challenge because:

* Both the node api docs and manual docs use 'filter' for both updateOne and findOneAndUpdate 
* Both the node api docs and manual docs use 'query' for both find and findOne


Sources:
---------
updateOne:
* https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#updateOne
* https://docs.mongodb.com/manual/reference/method/db.collection.updateOne/

findOneAndUpdate:
* https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#findOneAndUpdate
* https://docs.mongodb.com/manual/reference/method/db.collection.findOneAndUpdate/

find:
* https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#find
* https://docs.mongodb.com/manual/reference/method/db.collection.find/

findOne:
* https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#findOne
* https://docs.mongodb.com/manual/reference/method/db.collection.findOne/